src/components/Settings/RecorderSettings.tsx

```diff
@@ -60,6 +74,21 @@ export const RecorderSettings = () => {
         hint="Google Chrome needs to be installed on your machine for the recording functionality to work"
         disabled={recorder.detectBrowserPath}
       />
+
+      {!recorder.detectBrowserPath &&
+        recorder.browserPath !== '' &&
+        !isValidPath(recorder.browserPath?.toLocaleLowerCase()) && (
+          <Callout.Root color="amber">
+            <Callout.Icon>
+              <ExclamationTriangleIcon />
+            </Callout.Icon>
+
+            <Callout.Text>
+              The selected executable doesn&apos;t appear to be compatible.
+              Please select the correct executable for Chrome or Chromium.
+            </Callout.Text>
```

Comment:
This message does not really detect that the selected browser is correct. In practice you could rename the Firefox executable to `Chrome` and it would bypass it, but it handles the most common scenarios and warns the user about incompatibilities.